tests/io/test.directory.js

```diff
@@ -198,3 +201,19 @@ describe('Directory.getFileAsString()', function() {
   });
 
 });
+
+
+describe('Directory.getChunkAsBuffer()', function() {
+
+  it('should get a buffer', () => {
+    var myDirectory = new Directory('tests/fixtures/io/');
+    return myDirectory.getFiles()
+      .then(() => {
+        return myDirectory.getChunkAsBuffer('dir2/dir3/file3.txt', 2);
```

Comment:
As an aside - with fat arrow function if it's a single line return like this you can forgo the braces and the return and just have the expression. However, this is clear and explicit so it's fine as is.